frontend/src/Routes/User/UserProfile/RateUser.jsx

```diff
@@ -0,0 +1,52 @@
+import React, {useEffect, useState} from 'react';
+import {Avatar, Card, Comment, List, Rate, Skeleton, Text, Title} from "antd";
+
+import {connect} from '../../../Store';
+import avatarUrl from "../../../Helpers/avatar";
+import moment from "moment";
+import CommentInput from "./CommentInput";
+import {commentsGet} from "../../../Api/Comment";
+import delay from "../../../Helpers/delay";
+import {addRating, getRatingBetween, updateRating} from "../../../Api/Rate";
+
+
+const RateUser = ({ store: { userId }, otherUserId, onSubmit }) => {
+
+  const [value, setValue] = useState(0);
+  const [disabled, setDisabled] = useState(true);
+
+  useEffect(() => {
+    setDisabled(true);
+    getRatingBetween({receiverId: otherUserId, senderId: userId })
+      .then(res => {
+        const {data = {}} = res.data || {};
+        setValue(data);
+        setDisabled(false);
+      }).catch();
```

Comment:
I'm guessing this is the line where you need help.

maybe  you can change these lines to
```
.then(res => {
        const {data = {}} = res.data || {};
        setValue(data);
        setDisabled(false);
      }).catch((e) => console.log('catched error', e));
```
to first check out what error is being thrown there.
Then, you can use the error object to calculate the initial state.
axios errors usually include a response within it, (ie: e.response.data...)